src/components/Navbar.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faMicrophoneAlt, faUserCog, faChevronLeft } from '@fortawesome/free-solid-svg-icons';
+import { useHistory } from 'react-router-dom';
+import '../styles/Navbar.css';
+
+const Navbar = () => {
+  const history = useHistory();
+
+  const handleArrowLeftClick = () => {
+    history.push('/');
+  };
+
+  const handleArrowLeftKeyDown = (event) => {
+    if (event.key === 'Enter' || event.key === ' ') {
+      history.push('/');
+    }
+  };
```

Comment:
- [ ] [OPTIONAL] Good job implementing your re-routing with `useHistory` for ease of accessibility but I think it's more up-to-date to use `useNavigate` for the redirection or re-routing. Using it will be like:
``` 
import { useNavigate } from 'react-router-dom';
const navigate = useNavigate();
 const handleArrowLeftClick = () => {
    navigate('/');
  };
```

Reply:
Thank for the feedback. I have implemented your suggestion!